src/components/Card/CardContainer.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import { Button } from 'components/Common';
+import { CardCon, CardImage, CardImageContainer, CardTextContainer } from './CardUtils';
+
+const CardContainer = () => {
+  return (
+    <CardCon>
+      <CardImageContainer>
+        <CardImage src="#" alt="Image By AI" />
+        <CardImage src="#" alt="Image By AI" />
+        <CardImage src="#" alt="Image By AI" />
+        <CardImage src="#" alt="Image By AI" />
+        <CardImage src="#" alt="Image By AI" />
+      </CardImageContainer>
+      <CardTextContainer>
+        <div>
+          <p style={{ fontSize: '1.2rem', margin: '0 0 1rem 0' }}>CafeName</p>
```

Comment:
Is there any reason you used Inline-CSS?

Reply:
I used Inline-CSS, because I needed to check directly and those are minor details, so I thought it would be cleaner in this way.